### DISCUSSION TITLE
Sunil Pai - Partykit.io

### DISCUSSION THREAD

**Discussion by just-be-dev** [devtools-fm/devtools.fm#140] [Announcements]
## Background
- [twitter.com/threepointone](https://twitter.com/threepointone)
- [github.com/threepointone](https://github.com/threepointone)
- https://partykit.io/
- https://github.com/sponsors/threepointone (A good overview of his work)

Tweets on partykit
- https://twitter.com/partykit_io/status/1612239829093289991/photo/1
- https://twitter.com/partykit_io/status/1615436278958284825/photo/2

## Themes
- partykit / websockets
- cloudflare / edge compute

## Intro
Andrew: Hello, welcome to the devtools.fm podcast. This is a podcast about developer tools and the people who make them. I'm Andrew and this is my co-host Justin.

Justin: Hey everyone! Our guest today is Sunil Pai. Sunil worked on the React team at Facebook, created the css-in-js library glamor, built cloudflare's CLI tool wrangler, helped create babel macros, amongst many other contributions. He's now branching off on his own endeavor building out partykit which we're excited to talk about today. Before we dig into that though, Sunil, would you like to tell our listeners a little more about yourself?

### React

- [x] What parts of the react codebase did you work on when you were on the core team? 
- [x] How do you feel about where react is today and its future direction?
- [x] What's your biggest takeaway from working on the project?

### Edge Computing

- [x] You worked at cloudflare for a bit and have been a large proponent of edge computing.
- [x] For those who don't know what is edge computing and how is it different/better
- [x] What about this space excites you the most?
- [x] What were some of your favorite features you worked on at cloudflare?
- [x] How do you think edge computing will grow and evolve in the coming years?

### PartyKit

- [x] Can you give us an overview of what PartyKit is?
- [x] What is start shaped architecture?
- [x] What does working with party kit look like? How easy is it to integrate into my existing app?
- [x] What inspired you to start a business around this problem?
- [x] Who do you think your primary competitors are?
- [x] How do you support websockets on edge compute?

### Future

- [x] What do you think the the future of web apps is?

**Comment by just-be-dev** [comment]
Tooltips:
- https://github.com/shobrook/adrenaline
- https://copilot.money/

**Comment by hipstersmoothie** [comment]
- https://github.com/bvaughn/react-resizable-panels
- https://tinybase.org

**Comment by threepointone** [comment]
- https://maggieappleton.com/
- https://open.spotify.com/album/6Per97deaWqrJlKQNX8RGK?si=K4eOviwfT2mqPWgyxdldsw
